content/correspondence/reference/API endpoints/initialize/baseCorrespondenceExt/_index.en.md

```diff
@@ -0,0 +1,99 @@
+---
+title: BaseCorrespondenceExt
+linktitle: BaseCorrespondenceExt
+description: Description of the properties
+
+weight: 60
+toc: true
+---
+
+### resourceId
+
+Type: _string_
+
+The ID for the resource associated with the correspondence
+
+### sender
+
+Type: _string_
+
+The sender of the correspondence.
+
+### sendersReference
+
+Type: _string_
+
+Used by senders and receiver to identify a specific Correspondence using external identification methods.
+
+### messageSender
+
+Type: _string_
+
+An alternative name for the sender of the correspondence. The name will be displayed instead of the organization name.
+
+### content
+
+Type: [InitializeCorrespondenceContentExt](https://github.com/Altinn/altinn-correspondence/blob/main/src/Altinn.Correspondence.API/Models/InitializeCorrespondenceContentExt.cs)
+
+The correspondence content. Contains information about the title, body, etc.
+
+### requestedPublishTime
+
+Type: _DateTimeOffset_
+
+When the correspondence should become visible to the recipient.
```

Comment:
Consider adding:
If left NULL during initialization, the Correspondence is published immediatly.